app/config/Migrations/20231025052003_InsertData.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare(strict_types=1);
+
+use Migrations\AbstractMigration;
+
+class InsertData extends AbstractMigration
+{
+    public function change()
```

Comment:
About the `change` method usage, please kindly refer the https://book.cakephp.org/phinx/0/en/migrations.html#the-change-method

> The following actions are reversible when done through the Table API in Phinx, and will be automatically reversed:
>   - Creating a table
>   - Renaming a table
>   - Adding a column
>   - Renaming a column
>   - Adding an index
>   - Adding a foreign key

We perform the insert data here. Since the migration can be revert, does execute the migration rollback insert the data again or any unexpected behavior?

Please check.

Reply:
Usually, all the migrations should be in the up method, and the reversal should be in the down method.
Answering your question, if we run a change method in a revert or rollback, the inserted data won't be removed.